src/commands/Minion/blastfurnace.ts

```diff
@@ -0,0 +1,181 @@
+import { CommandStore, KlasaMessage, KlasaUser } from 'klasa';
+import { Bank } from 'oldschooljs';
+
+import { Activity, Time } from '../../lib/constants';
+import { hasGracefulEquipped } from '../../lib/gear/functions/hasGracefulEquipped';
+import { minionNotBusy, requiresMinion } from '../../lib/minions/decorators';
+import { UserSettings } from '../../lib/settings/types/UserSettings';
+import Smithing from '../../lib/skilling/skills/smithing';
+import { SkillsEnum } from '../../lib/skilling/types';
+import { BotCommand } from '../../lib/structures/BotCommand';
+import { ItemBank } from '../../lib/types';
+import { BlastFurnaceActivityTaskOptions } from '../../lib/types/minions';
+import {
+	bankHasItem,
+	formatDuration,
+	itemID,
+	itemNameFromID,
+	removeItemFromBank,
+	skillsMeetRequirements,
+	stringMatches
+} from '../../lib/util';
+import addSubTaskToActivityTask from '../../lib/util/addSubTaskToActivityTask';
+
+export function hasSkillRequirements(user: KlasaUser): boolean {
+	return skillsMeetRequirements(user.rawSkills, {
+		crafting: 12,
+		firemaking: 16,
+		magic: 33,
+		mining: 50,
+		smithing: 20,
+		thieving: 13
+	});
+}
```

Comment:
This is a very generic/vague function to be exporting - it should be not exported, or be given a descriptive name.